ntoskrnl/ke/amd64/cpu.c

```diff
@@ -10,8 +10,10 @@
 /* INCLUDES *****************************************************************/
 
 #include <ntoskrnl.h>
+
```

Comment:
Doesn't look like a missing `\n` to me. Please stop injecting style changes into a PR that has a totally different purpose.